.github/workflows/CI.yml

```diff
@@ -27,15 +27,40 @@ jobs:
 
       - name: Run tests
         run: |
-          go test -integration -v ./...
-          go test -testserver -v ./...
-          go test -integration -testserver -v ./...
-          go test -integration -allocator -v ./...
-          go test -testserver -allocator -v ./...
-          go test -integration -testserver -allocator -v ./...
-          go test -race -integration -v ./...
-          go test -race -testserver -v ./...
-          go test -race -integration -testserver -v ./...
-          go test -race -integration -allocator -v ./...
-          go test -race -testserver -allocator -v ./...
-          go test -race -integration -allocator -testserver -v ./...
+          make integration
+          make integration_w_race
+
+  run-tests-other-archs:
+    name: Run tests on other archs
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - arch: aarch64
+            distro: ubuntu20.04
+            go-arch: arm64
+          - arch: ppc64le
+            distro: ubuntu20.04
+            go-arch: ppc64le
+          - arch: armv7
+            distro: ubuntu20.04
+            go-arch: armv6l
+    steps:
+      - uses: actions/checkout@v2
+      - uses: uraimo/run-on-arch-action@v2.0.8
+        name: Run tests
+        with:
+          arch: ${{ matrix.arch }}
+          distro: ${{ matrix.distro }}
+          shell: /bin/bash
+
+          install: |
+            apt-get update -q -y
+            apt-get install -q -y curl gcc make openssh-sftp-server
+            GO_VERSION=$(curl https://golang.org/VERSION?m=text)
```

Comment:
So, we run the integration tests for 1.14 and 1.15 for x86_64, and then run the other architectures only on the most recent version? (Which would jump automatically at 1.16?)

Reply:
Yes, correct